service/static/items.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Wishlist RESTful Service</title>
```

Comment:
Maybe say `Wishlist RESTful Service (Items)`

Reply:
> List all wishlists and Items should work without any params. I get an error when I try that.
> 
> ![image](https://user-images.githubusercontent.com/21155466/98770599-97a01880-23b0-11eb-950f-91c9e917e7be.png)

I just finished UI and wrote some samples in features and restapi.js, but not real interaction. Let's leave the functions to other stories.

Reply:
> Maybe say `Wishlist RESTful Service (Items)`

Done